cJSON.c

```diff
@@ -1660,6 +1660,11 @@ static cJSON_bool parse_object(cJSON * const item, parse_buffer * const input_bu
             current_item = new_item;
         }
 
+        if (input_buffer->offset+1 >= input_buffer->length)
```

Comment:
Maybe we can implement this with `cannot_access_at_index` to be more consistent with existing code styles?
```
if (cannot_access_at_index(input_buffer, 1))
```

Reply:
Good suggestion. Done.